base/commands/viridian/viridian_cluster_get.go

```diff
@@ -102,6 +102,11 @@ func (cm ClusterGetCmd) Exec(ctx context.Context, ec plug.ExecContext) error {
 				Type:  serialization.TypeStringArray,
 				Value: regionTitleSlice(c.Regions),
 			},
+			output.Column{
+				Name:  "Dev Mode",
+				Type:  serialization.TypeBool,
+				Value: c.ClusterType.DevMode,
```

Comment:
How about setting the name to `Cluster Type` and `Value` to one of `Production` or `Development`?